dahu/core/app/scripts/layouts/workspace.js

```diff
@@ -6,10 +6,15 @@ define([
     'underscore',
     'handlebars',
     'backbone.marionette',
+    // views
     'views/workspace/screen',
     'views/workspace/actions',
+    'views/workspace/note',
+    // template
     'text!templates/layouts/workspace.html'
-], function(_, Handlebars, Marionette, screenView, actionsView, workspaceTemplate){
+], function(_, Handlebars, Marionette, 
+        screenView, actionsView, noteView, // views
+        workspaceTemplate){ // template
```

Comment:
Please respect the format:

define('', [
    'patcher',
    // libraries
    'backbone',
    // modules
    'modules/kernel/SCI',
    'modules/screencast',
    // models
    'models/screencast'
], function(
    Patcher,
    Backbone,
    Kernel, Screencast,
    ScreencastModel
) {